server/src/main/java/org/elasticsearch/action/TaskOperationFailure.java

```diff
@@ -95,16 +118,20 @@ public Exception getCause() {
 
     @Override
     public String toString() {
-        return "[" + nodeId + "][" + taskId + "] failed, reason [" + getReason() + "]";
```

Comment:
I did not find directly usages of `TaskOperationFailure::toString` and for me seems that nobody should rely on `toString` until it's specifically mentioned in JavaDoc for this method, maybe I live in a parallel universe :)

Reply:
No, I agree. Nevertheless we might want to play it save, but I will get another opinion on this.

Reply:
I think that this change is not necessary, and changing it is a risk. It could just be that this is logged somewhere and the current toString output makes sense the way it is, maybe even more readable than the json output.